### PR TITLE
SMS Game Signup form

### PR DESCRIPTION
**Pull request by aaronschachter**
@angaither Please review.

Continues development on SMS Game Campaign type.
- Adds form elements for alpha phone, beta phones
- Creates signup for auth user upon submit
- Redirects user to reportback confirmation page after submit
- Alters confirmation page logic to display for anon users if node is a sms game. 
- Outputs starter statement field into tpl

### CODE REVIEW

File: lib/modules/dosomething/dosomething_signup/dosomething_signup.forms.inc [hunk 1]
```diff
@@ -31,3 +57,20 @@ function dosomething_signup_sms_game_form($form, &$form_state) {
...
   return $form;
 }
+
+/**
+ * Form submission handler for dosomething_signup_alpha_beta_form().
+ *
+ * Redirects user to the reportback confirmation page for given node.
+ */
+function dosomething_signup_alpha_beta_form_submit($form, &$form_state) {
+  $values = $form_state['values'];
```
Comment (angaither) [inline lib/modules/dosomething/dosomething_signup/dosomething_signup.forms.inc:122]: need to make sure to check these phone numbers against the regex https://github.com/DoSomething/dosomething/blob/dev/lib/modules/dosomething/dosomething_user/dosomething_user.module#L110

Also we have something in the ds_user module that checks if that phone number is already on an account, do we want/need that for this?

Comment (aaronschachter) [inline lib/modules/dosomething/dosomething_signup/dosomething_signup.forms.inc:122]: Yeah that's a good point re: rege, I will add that in.

Asked @mikefantini about this and I think for right now, we don't need to worry about if the # belongs to a different account.

**Comment by aaronschachter** [comment]
@angaither Updated with a server side validation function.

**Comment by angaither** [comment]
:+1:

**Pull request merged**